Gems/QtForPython/Code/Source/QtForPythonModule.cpp

```diff
@@ -8,25 +8,31 @@
 
 #include <AzCore/Memory/SystemAllocator.h>
 #include <AzCore/Module/Module.h>
+#include <AzCore/Debug/Trace.h>
```

Comment:
why does this cpp need <AzCore/Debug/Trace.h> ?

Reply:
Oops, that was for AZ_Error which use to be in this file before I refactored it out